src/handson/Table.js

```diff
@@ -0,0 +1,336 @@
+"use strict";
+(function(root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "handsontable", "../common/Widget", "../common/HTMLWidget", "../common/PropertyExt", "../common/Utility", "css!handsontable","css!./Table"], factory);
+    } else {
+        root.handson_Table = factory(root.d3, root.Handsontable, root.common_Widget, root.common_HTMLWidget, root.common_PropertyExt, root.common_Utility);
+    }
+}(this, function(d3, Handsontable, Widget, HTMLWidget, PropertyExt, Utility) {
+
+    function Column(owner) {
+        PropertyExt.call(this);
+        this._owner = owner;
+    }
+    Column.prototype = Object.create(PropertyExt.prototype);
+    Column.prototype.constructor = Column;
+    Column.prototype._class += " handson_Table.Column";
+
+    Column.prototype.publish("label", "", "set", "Only show column properties of the table", function() { return this._owner ? this._owner.columns() : [];}, {optional: true });
+    Column.prototype.publish("minRange", null, "number", "Minimum value of a handsontable", function() { return this._owner ? this._owner.columns() : [];});
+    Column.prototype.publish("maxRange", null, "number", "Maximum value of a handsontable", function() { return this._owner ? this._owner.columns() : [];});
```

Comment:
```function () { return this._owner ? this._owner.columns() : []; }``` is invalid here (this param is only used for "set" PP types) - it should be null.